cmd/mssqldef/mssqldef_test.go

```diff
@@ -24,10 +24,17 @@ func TestMssqldefColumnLiteral(t *testing.T) {
 	resetTestDatabase()
 
 	createTable := stripHeredoc(`
-		CREATE TABLE users (
-		  id integer NOT NULL,
-		  name text,
-		  age integer
+		CREATE TABLE v (
+		  v_integer integer NOT NULL,
+		  v_text text,
+		  v_smallmoney smallmoney,
+			v_money money,
+			v_datetimeoffset datetimeoffset(1),
+			v_datetime2 datetime2,
+			v_smalldatetime smalldatetime,
+			v_nchar nchar(30),
+			v_nvarchar nvarchar(30),
+			v_ntext ntext
```

Comment:
As you can see in this diff on GitHub, the indentation is inconsistent. Please consider checking your code before submitting a PR next time.
```suggestion
		  v_integer integer NOT NULL,
		  v_text text,
		  v_smallmoney smallmoney,
		  v_money money,
		  v_datetimeoffset datetimeoffset(1),
		  v_datetime2 datetime2,
		  v_smalldatetime smalldatetime,
		  v_nchar nchar(30),
		  v_nvarchar nvarchar(30),
		  v_ntext ntext
```

Reply:
Sorry for my carelessness, I'll be careful next time.